tools/cli/commands/changelog.js

```diff
@@ -729,62 +742,67 @@ async function promptChangelog( argv, needChangelog, types ) {
 		.stdout.toString()
 		.trim()
 		.replace( /\//g, '-' );
-	const maxLength = Object.keys( types ).reduce( ( a, v ) => ( v.length > a ? v.length : a ), 0 );
-	const choices = Object.entries( types ).map( ( [ value, name ] ) => ( {
-		value,
-		name: `[${ value.padEnd( maxLength, ' ' ) }] ${ name }`,
-	} ) );
-	const commands = await inquirer.prompt( [
-		{
-			type: 'string',
-			name: 'changelogName',
-			message: 'Name your changelog file:',
-			default: gitBranch,
-			validate: input => {
-				const fileExists = doesFilenameExist( input, needChangelog );
-				if ( fileExists ) {
-					return 'Please choose another file name, or delete the file manually.';
-				}
-				return true;
-			},
+	const significanceChoices = {
+		patch: '[patch] Backwards-compatible bug fixes.',
+		minor: '[minor] Added (or deprecated) functionality in a backwards-compatible manner.',
+		major: '[major] Broke backwards compatibility in some way.',
+	};
+	const changelogName = await inquirer.prompt( {
+		type: 'string',
+		name: 'changelogName',
+		message: 'Name your changelog file:',
+		default: gitBranch,
+		validate: input => {
+			const fileExists = doesFilenameExist( input, needChangelog );
+			if ( fileExists ) {
+				return 'Please choose another file name, or delete the file manually.';
+			}
+			return true;
 		},
-		{
-			type: 'list',
-			name: 'significance',
-			message: 'Significance of the change, in the style of semantic versioning.',
-			choices: [
-				{
-					value: 'patch',
-					name: '[patch] Backwards-compatible bug fixes.',
-				},
-				{
-					value: 'minor',
-					name: '[minor] Added (or deprecated) functionality in a backwards-compatible manner.',
-				},
-				{
-					value: 'major',
-					name: '[major] Broke backwards compatibility in some way.',
-				},
-			],
+	} );
+	const significance = await autoComplete( {
+		message: 'What is the significance of the change?',
+		source: async input => {
+			const significanceArray = Object.keys( significanceChoices );
+			const filteredSignificance = await filterInput( significanceArray, input );
+			return filteredSignificance.map( choice => {
+				return {
+					value: choice,
+					description: significanceChoices[ choice ]
+						? significanceChoices[ choice ]
+						: 'Unknown significance',
+				};
+			} );
 		},
-		{
-			type: 'list',
-			name: 'type',
-			message: 'Type of change.',
-			choices: choices,
+	} );
+
+	const type = await autoComplete( {
+		message: 'What type of change?',
+		source: async input => {
+			const typesArray = Object.keys( types );
+			const filteredType = await filterInput( typesArray, input );
+			return filteredType.map( choice => {
+				return {
+					value: choice,
+					description: types[ choice ] ? types[ choice ] : 'Unknown type',
```

Comment:
Same.
```suggestion
					name: types[ choice ],
```
In this case we'd also want to bring back the `[value]` indicator at the start of the lines ([lines 732–736 from the base version](https://github.com/Automattic/jetpack/blob/44ce10fb1eb5bf6a0ebaa7a936d4868de4c81d1a/tools/cli/commands/changelog.js#L732-L736)), or else have the autocompletion search against the displayed text in addition to the value.